Formula/hueadm.rb

```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class Hueadm < Formula
+  desc "Command-line management interface to phillips hue"
+  homepage "https://github.com/bahamas10/hueadm"
+  url "https://registry.npmjs.org/hueadm/-/hueadm-1.2.0.tgz"
+  sha256 "8b5936a84f112ce1d1ca9a2a917223755413e3dfa3854088fb4f2c01e2470e72"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system "#{libexec}/bin/hueadm", "search"
```

Comment:
```suggestion
    system "#{bin}/hueadm", "search"
```
We should test it from its install location to make sure it's working under normal use. Can we do an `assert_match`  or `assert_equal` here to check the output?

Reply:
Not with that command. Unfortunately, the output of any *meaningful* command is completely dependent on the home/network environment. Could just do `hueadm -v` and check the version, but it seemed like trying an actual search without error was a better test. Happy to take any suggestion, though.

Reply:
So no substring of the output of this command is common across systems? Not even something like a `Search Results:`, or similar?

Reply:
Ah, actually, yes... the first line should always be "IP". Any examples of that sort of test?

Reply:
Then you should be able to do
```ruby
assert_match "IP", shell_output("#{bin}/hueadm search")
```
You may need to redirect stderr if that's where your output goes.

Reply:
Added that assertion. No redirect needed - output goes to stderr